### PR TITLE
Content update

### PR DESCRIPTION
**Pull request by anonymoussprocket**
- phrasing
- links

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Harpoon
 
-Harpoon is a staking dashboard for Tezos built using ConseilJS. Harpoon collects metrics about each baker on the Tezos network and allows users to search for and evaluate bakers based off of these performance stats. This data includes blocks baked, missed, and stolen, stake relative to the rest of the network, as well as a letter grade based on a normalized scoring system. Harpoon also allows for a user to view and audit the rewards won/payed out by bakers.
+Harpoon is a staking dashboard for the Tezos blockchain built using [ConseilJS](https://github.com/Cryptonomic/ConseilJS). Harpoon collects metrics about each baker on the Tezos network and allows users to search for and evaluate bakers based on these performance stats. The data includes blocks baked, missed, and stolen, stake relative to the rest of the network, as well as a letter grade based on a normalized scoring system. Harpoon also enables baker auditing by showing the rewards won and payed out. [Cryptonomic](https://twitter.com/cryptonomictech) has a public deployment of [Harpoon](https://harpoon.arronax.io/) under the [Arronax](https://harpoon.arronax.io/) umbrella of blockchain analytics products.
```
Comment (PrathamRawat) [inline README.md:3]: > If the verb pay is used in a nautical sense, the correct form is payed.

Wow I didn't know we'd stick to our nautical theme this hard
Should be 'paid' instead

File: README.md [hunk 2]
```diff
@@ -24,18 +24,18 @@ After configuration, Harpoon can be run through [Docker](https://docs.docker.com
...
 
 ## How It Works
 
-Much of the functionality of Harpoon is achieved through client-side javascript using ConsielJS, such as the stats on the top left panel that provide an outline for the baker's performance in the last cycle. However, for information that is data intensive or not available through Consiel, a database is maintained and kept up to date by server side python scripts. The following sections provide an overview for some of the less conventional pieces of information are collected.
+Much of the functionality of Harpoon is achieved through client-side javascript using [ConsielJS](https://www.npmjs.com/package/conseiljs), such as the stats on the top left panel that provide an outline for the baker's performance in the last cycle. However, for information that is data intensive or not available through [Conseil](https://github.com/Cryptonomic/Conseil), a database is maintained and kept up-to-date with [additional indexing scripts](https://github.com/Cryptonomic/Harpoon/tree/master/server) in Python. The following sections provide an overview for some of the less conventional pieces of information that are collected.
```
Comment (PrathamRawat) [inline README.md:34]: ConsielJS should be ConseilJS

File: README.md [hunk 3]
```diff
@@ -24,18 +24,18 @@ After configuration, Harpoon can be run through [Docker](https://docs.docker.com
 docker-compose up --build
 ```
 
-Doing so will result in an endpoint at http://127.0.0.1:<web_server_port>, where `web_server_port` was the values specified during configuration. A Postgres database is also accessible through the corresponding port specified during configuration a well.
+Doing so will result in an endpoint at http://127.0.0.1:<web_server_port>, where `web_server_port` was the value specified during configuration. A Postgres database is also accessible through the corresponding port specified during configuration a well.
```
Comment (PrathamRawat) [inline README.md:27]: I think we should display "http://127.0.0.1:<web_server_port>" in code formatting?

**Review by PrathamRawat — COMMENTED** [review]
Some very minor errors in the writing that don't necessarily **_need_** to be fixed, but would be nice if they were

**Review by Taaseen-Ali — APPROVED** [review]

**Review by vishakh — APPROVED** [review]

**Pull request merged**